### PR TITLE
fix validator anchor

### LINKED ISSUE #2323: Add anchor to validator website
**Is your feature request related to a problem? Please describe.**

The website from the validator page doesn't contain an anchor to go to the page.

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #2323

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -116,7 +116,15 @@
             <dl class="info_dl">
               <dt>Website</dt>
               <dd>
-                {{ translateEmptyDescription(validator.description.website) }}
+                <a
+                  id="validator-website"
+                  :href="validator.description.website ?
```
Comment (sabau) [inline app/src/renderer/components/staking/PageValidator.vue]: It's on 2 lines XD I wanted to add commit suggestion :)

we could write `validator.description.website || null` or just `validator.description.website`

Comment (sabau) [reply app/src/renderer/components/staking/PageValidator.vue]: Just extra extra, if you want to disable the link if the website is not there
```
:disabled="!validator.description.website"
```

Comment (fedekunze) [reply app/src/renderer/components/staking/PageValidator.vue]: addressed on pair programming

File: app/src/renderer/components/staking/PageValidator.vue [hunk 2]
```diff
@@ -288,6 +298,11 @@ export default {
       // status: active
       return `green`
     },
+    // empty descriptions have a strange '[do-not-modify]' value which we don't want to show
+    website() {
+      const url = this.validator.description.website
```
Comment (sabau) [inline app/src/renderer/components/staking/PageValidator.vue]: minor: we could pass the parameter straight to the function

**Pull request merged**